test_suite/package/test_otel.py

```diff
@@ -0,0 +1,132 @@
+import pytest
+import logging
+import time
+import json
+
+from test_suite.generic.test_generic import TestsSubscriptionManager
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def check_instance_status(request, instance_data, host):
+    instance_id = instance_data['instance_id']
+    command_to_run = [
+        'aws', 'ec2', 'describe-instance-status',
+        '--instance-ids', instance_id
+    ]
+
+    while True:
```

Comment:
Add a safety net/timeout to this while, it can enter into an infinite loop.

Reply:
ACK